cmd/zoekt-sourcegraph-indexserver/queue.go

```diff
@@ -47,11 +55,15 @@ type queueItem struct {
 //
 // * We rather index a repo sooner if we know the commit is stale.
 // * The order of repos returned by Sourcegraph API are ordered by importance.
+// GLEE: Is the above statement true?
```

Comment:
@keegancsmith this is the comment I mentioned previously. I could be misinterpreting it but I interpret it as some prioritization logic taking place on sourcegraph side (or sourcegraph [client](https://sourcegraph.com/github.com/sourcegraph/zoekt/-/blob/cmd/zoekt-sourcegraph-indexserver/sg.go?L132:29), but that didn't seem to be the case to me).

Reply:
Also if you see `GLEE` code comments then I'll remove before merging PR.

Reply:
You are correct, this is no longer true. We used to do that a very long time ago.